README.md

```diff
@@ -7,10 +7,10 @@ In this respository is a Python version of the source code, but there is also a
 ## Requirements
 
 Either of the following:
-* Any Python from 2.7 and beyond, with only the standard library and [MicroQiskit](https://github.com/qiskit-community/MicroQiskit);
 * Python 3.7 and beyond with Qiskit, NumPy, SciPy and PIL.
```

Comment:
```suggestion
* Python 3.9 and beyond with Qiskit, NumPy, SciPy and PIL.
```
Python 3.7 has no support and 3.8 is reaching end-of-life soon.  https://endoflife.date/python